dev-requirements.txt.py

```diff
@@ -0,0 +1 @@
+pytest>=5.3.1,!=5.4.0
```

Comment:
I think this file is here by mistake?

Reply:
I put the file there to match the representer config here: https://github.com/exercism/python-representer.  The CI needs `pytest` for running tests, but the analyzer doesn't need `pytest` for its "normal" functioning.  Should I be adding `pytest` to the main `requirements.txt` instead?  What would be your suggested way of handling needing `pytest` for CI/dev testing?

Reply:
UGH.  Thought I got rid of the `.py` version.  🤦🏽   Will delete.  Thanks!